src/components/shared/GoogleAutoComplete/GoogleAutoComplete.tsx

```diff
@@ -31,30 +28,22 @@ class GoogleAutoComplete extends React.Component<Props, any> {
     this.autocomplete = new google.maps.places.Autocomplete(
       this.props.inputRef.current,
       {"types": ["(regions)"]}
-    )
+    );
     if (this.autocomplete.setFields) this.autocomplete.setFields(['geometry', 'name']);
     this.autocomplete.addListener('place_changed', this.handlePlaceChanged);
   }
 
   handlePlaceChanged = () => {
-    this.props.onPlaceChange(this.autocomplete.getPlace());
+    const input= document.getElementById('locationQuery') as HTMLInputElement;
```

Comment:
did we want this be a ref as per @woeltjen 's comment?

Reply:
good catch, fixing now